### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by lmroces** [audiamus/BookLibConnect#84] [General]
I could not find a "thank you" thread, but I just wanted to thank you for both BokkLibConnect and for AaxAudioConverter. I don't have a smartphone or any other similar device to download to my pc to upload the books to an iPod classic (yes, ancient but works), except to download to an android simulator on my pc, and not finding where the books were downloaded, i used to have to then record the audiobook as it played via bluestacks! You are an angel!!!

**Comment by audiamus** [comment]
Thank you for your kind feedback. You are very welcome.

**Comment by Grokka** [comment]
Another for the should be very yuge Thanks! thread. 
Vaguely sure how I made it work after the setup, simply: 
DL within BookLibConnect, convert in AAX, boom there it is.   
Worked flawlessly. And I have No Idea what I am doing.
Fantastic.
I really appreciate this. I dont pirate things, but I also refuse to run "apps". 
And I really dislike being logged in here or there to do any whatever. 
Whose machine is this, whose life is this, anyway? 
*
Its not a minor question. Its a very big deal.
Work like yours renews my faith, and my freedom. 
Today's everyday heros, you are
Thank you!

**Comment by IlluminatorX** [comment]
In my oppinion this is is a great piece of sw. Both the downloader and the converter. Thank you, Muchas  gracias etc ;)
